src/main/java/htsjdk/tribble/util/ParsingUtils.java

```diff
@@ -429,10 +430,16 @@ public static boolean resourceExists(String resource) throws IOException{
      * @return
      */
```

Comment:
Update the javadoc here to point to the new method instead of the deprecated.

Reply:
The deprecated (reflection) method has been removed.